generics/constants/api-responses.js

```
@@ -286,6 +286,12 @@ module.exports = {
 	ADMIN_TOKEN_MISSING_CODE: 'ERR_ADMIN_TOKEN_MISSING',
 	ADMIN_TOKEN_MISSING_MESSAGE: 'Admin token is missing',
 	SOLUTION_NOT_ALLOWED_TO_BE_CONSUMED: 'This solution is not allowed to be consumed privately',
+	SOLUTION_NOT_FOUND: 'Solution not found',
```

ESLint

[error] Duplicate key 'SOLUTION_NOT_FOUND'.

(no-dupe-keys)